client/src/Components/TaskEstimatedBox/TaskEstimatedBox.jsx

```diff
@@ -1,29 +1,48 @@
-import React, { useState } from 'react';
-import { Modal, Box } from '@material-ui/core';
+import React, { useState, forwardRef } from 'react';
+import { useBackend } from 'hooks/useBackend';
+import { Dialog, DialogContent, DialogTitle } from '@material-ui/core';
 import TaskEstimatedBoxStyles from './TaskEstimatedBoxStyles';
-import TaskEstimatedChart from 'Components/TaskEstimatedChart';
-import TaskEstimationElement from 'Components/TaskEstimationElement';
+import TaskEstimatedChart from 'Components/TaskEstimatedChart/TaskEstimatedChart';
+import TaskEstimationElement from 'Components/TaskEstimationElement/TaskEstimationElement';
+import { IconButton } from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
+import { useSocket } from 'socketio-hooks';
 
 const TaskEstimatedBox = () => {
-  const classes = TaskEstimatedBoxStyles();
   const [open, setOpen] = useState(false);
-
+  const [taskName, setTaskName] = useState();
+  const [userVoted, setUserVoted] = useState([]);
+  const [resultsAnalysis, setresultsAnalysis] = useState({});
+  const sendVotesClear = useBackend('CLEAR_VOTES');
```

Comment:
po zadeklarowaniu wszystkich hooków proponuje dawać linie odstępu, nie jest to żadna konwencja ale moim zdaniem lepiej się to czyta